src/components/Api.js

```diff
@@ -0,0 +1,169 @@
+import { renderLoading } from "../utils/utils.js";
+
+import { api } from "../pages/index.js";
+import {
+  profileEditForm,
+  cardsEditForm,
+  propicForm,
+} from "../utils/constants.js";
+
+export class Api {
+  constructor(options) {
+    this._options = options;
+  }
+
+  loadUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
+      headers: {
+        authorization: "f01264e8-2101-4ab7-b120-b09b55e63681",
+      },
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      return Promise.reject("Error: ${res.status}");
+    });
+  }
+
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        authorization: "f01264e8-2101-4ab7-b120-b09b55e63681",
+      },
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      return Promise.reject("Error: ${res.status}");
+    });
+  }
+
+  editProfile({ name, about }) {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
+      method: "PATCH",
+      headers: {
+        authorization: "f01264e8-2101-4ab7-b120-b09b55e63681",
+        "Content-Type": "application/json",
+      },
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)